GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -1160,7 +1147,9 @@ bool TextureCacheVulkan::GetCurrentTextureDebug(GPUDebugBuffer &buffer, int leve
 	TexCacheEntry *entry = nextTexture_;
 	ApplyTexture();
 
+	/*
 	// TODO: Centralize?
+	// TODO: Fix!
```

Comment:
I really rely on this when using the GE debugger so I think it'd be better to fix it before merging...

-[Unknown]

Reply:
Oops, yeah, was meaning to get back to that. These will be fixed before merge.